### DISCUSSION TITLE
About “pip install .”

### DISCUSSION THREAD

**Discussion by ooooppps** [deepmodeling/deepmd-kit#1439] [Q&A]
while installing deepmd from source code and running line "pip install .", the process aborted for servers cannot connect to the Internet. Terminal seems to run a "pip install" line with option"--ignore-installed". On this occasion, how can i do to make it run without "--ignored-installed"?
Error info:
ERROR: Command errored out with exit status 1:
   command: /usr/local/python3/bin/python3.7 /tmp/pip-standalone-pip-voc6yuo0/__env_pip__.zip/pip install --ignore-installed --no-user --prefix /tmp/pip-build-env-mk7c2gtq/overlay --no-warn-script-location --no-binary :none: --only-binary :none: -i https://pypi.org/simple -- setuptools setuptools_scm wheel scikit-build cmake ninja m2r

**Comment by njzjz** [comment]
Hi, you can follow [this stack overflow answer](https://stackoverflow.com/a/14447068) to install packages offline.
